templates/android/library/src/main/java/io/package/services/Service.kt.twig

```diff
@@ -38,6 +38,19 @@ class {{ service.name | caseUcfirst }}(client: Client) : Service(client) {
      * @return [{{ method | returnType(spec, sdk.namespace | caseDot) | raw }}]
     {%~ endif %}
      */
+    {%~ if method.deprecated %}
+    {%~ if method.since and method.replaceWith %}
+    @Deprecated(
+        message = "This API is deprecated since {{ method.since }} and uses outdated terminologies. Please use `{{ method.replaceWith | capitalizeFirst }}` instead.",
```

Comment:
Not necessary that every deprecation would be due to outdated terminology. We can use a generic message. Use the sdk label if needed so every method/endpoint has a specific message.

Reply:
Goes for all other templates as well.

Reply:
@ChiragAgg5k Agree, let's remove ` and uses outdated terminologies`